CorsixTH/Lua/objects/litter.lua

```diff
@@ -138,19 +137,23 @@ function Litter:anyLitter()
 end
 
 function Litter:afterLoad(old, new)
+  if old < 151 then
```

Comment:
New ``afterLoad()``s should be at the bottom. (ascending order)

Reply:
Done.

Code didn't have a hospital already, and below my addition it uses `self.hospital` without either introducing it or removing it in any existing afterLoad.
I wouldn't be surprised if the code just dies if you try loading old stuff like version 52.